packages/holocron/__tests__/moduleRegistry.spec.js

```diff
@@ -38,7 +38,7 @@ describe('moduleRegistry', () => {
     const GoodModule = () => null;
     registerModule('good-module', GoodModule);
     expect(getModules()).toMatchSnapshot();
-    expect(getModule('good-module')).toBe(GoodModule);
+    expect(getModule('good-module').displayName).toBe('WithHolocronErrorBoundary(GoodModule)');
```

Comment:
any better option than asserting on the `displayName`?

Reply:
Resolved in 291928d